features.md

```diff
@@ -55,6 +55,7 @@ specification.
 | `sync-xhr` | [XMLHttpRequest][xhr] | [Chrome 65](https://www.chromestatus.com/feature/5154875084111872) |
 | `usb` | [WebUSB][webusb] | Chrome 60 |
 | `web-share` | [Web Share API][web-share] | Chrome 86 |
+| `window-management`<sup>[5](#fn5)</sup> | [Window Management API][window-management] | [Chrome 111](https://chromestatus.com/feature/5146352391028736) |
```

Comment:
Let me know if this belongs in proposed, given that this is only implemented in Chromium.

Reply:
This looks right; it's published in the appropriate spec. A number of these are only implemented in Chromium